### ISSUE TITLE
Suggested tooltip text

### ISSUE THREAD

**Issue by dhimmel** [hetio/repurpose-frontend#9]
I'll use this issue to leave suggested tooltip text for the react tables

**Comment by dhimmel** [comment]
# Compounds tooltips

Tooltips for https://hetio.github.io/repurpose-frontend/?tab=compounds

> ![image](https://user-images.githubusercontent.com/1117703/62726149-849bdd00-b9e4-11e9-869d-be6f210bffb1.png)

- ID: DrugBank identifier for the compound
- Name: name of the compound
- Treats: the number of disease-modifying therapies for the compound, i.e. how many diseases it's known to treat
- Edges: the number of edges the compound has to other nodes in Hetionet v1.0, i.e. total network degree. In general, the more edges, the more Hetionet knows about a node.
- AUROC: measures the ability of the predictions to prioritize these known treatments. AUROC refers to the area under an ROC curve and represents the probability that a randomly selected treatment will have received a higher prediction than a randomly selected non-treatment.


Tooltips for https://hetio.github.io/repurpose-frontend/?tab=compounds&id=DB01048

>  ![image](https://user-images.githubusercontent.com/1117703/62726688-d09b5180-b9e5-11e9-90c1-9bbd0e4d3049.png)

- Name: name of the disease
- Prediction: the predicted probability that {Abacavir} treats the disease
- Compound Percentile: the prediction's percentile within all predictions for {Abacavir}
- Disease Percentile: the prediction's percentile within all predictions for the disease
- Cat: the category in PharmacotherapyDB v1.0: DM refers to a disease-modifying indication, SYM refers to a symptomatic indication, NOT refers to a non-indication.
- Trials: the number of trials investigating this compound–disease pair in ClinicalTrials.gov
- Neo4j Actions: Load additional information for the prediction via the Hetionet Neo4j Browser. Either open a new "browser" with the prediction query prefilled (you still must hit play). Or copy the "command" to your clipboard to paste into an already-open Hetionet Browser

**Comment by dhimmel** [comment]
## Repurpose text

Text for https://hetio.github.io/repurpose-frontend

Browse drug repurposing predictions from Project Rephetio, published in:

```html
<p><strong>Systematic integration of biomedical knowledge prioritizes drugs for repurposing</strong><br />
Daniel Scott Himmelstein, Antoine Lizee, Christine Hessler, Leo Brueggeman, Sabrina L Chen, Dexter Hadley, Ari Green, Pouya Khankhanian, Sergio E Baranzini<br />
<em>eLife</em> (2017-09-22) <a href="https://doi.org/cdfk">https://doi.org/cdfk</a><br />
DOI: <a href="https://doi.org/10.7554/elife.26726">10.7554/elife.26726</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28936969">28936969</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640425">PMC5640425</a></p>
```

These predictions are based on predicting treatment edges in Hetionet v1.0, an integrative network of biomedicine containing 2,250,197 relationships of 24 types. Our approach learns network patterns of drug efficacy, translating the paths between a compound and disease into a predicted probability of treatment.

Navigate to your compound or disease of interest to see all of its predictions.

## Tab tooltips

Compounds tab: select a compound and show predictions for all diseases
Diseases tab: select a disease and show predictions for all compounds
Metapaths: browse types of paths (metapaths) that connect compounds to disease and their ability to predict drug efficacy

**Comment by dhimmel** [comment]
## Repurpose metapath tooltips

tooltips for https://hetio.github.io/repurpose-frontend/?tab=metapaths (mostly taken from https://het.io/repurpose/metapaths.html):

+ Abbrev: shorthand notation for metapaths with uppercase metanode abbreviations and lowercase metaedge abbreviations
+ Metapath: full name of the metapath (type of path). Metanodes (node types) use Title Case.
+ Len: length, i.e. the number of metaedges composing the metapath
+ Δ AUROC: AUROC assesses how well DWPCs for this metapath discriminated treatments from non-treatments (in the all-features stage). Δ AUROC refers to the difference between the AUROCs on the real and permuted networks, and assesses whether the performance of the metapath's DWPCs exceeded the performance of the node degree information it captures.
- p-value: assesses whether permutation affected the AUROC, i.e. the significance of Δ AUROC. Note that several metapaths shown here provided little evidence that Δ AUROC ≠ 0 underscoring their poor ability to predict whether a compound treated a disease, after accounting for degree information.
+ Coef: the metapath's logistic regression coefficient. Metapaths removed in feature selection have missing coefficients whereas metapaths given zero-weight by the elastic net have coef. = 0.0.

**Comment by dhimmel** [comment]
## License text 

We'll want to do something like what is currently at https://het.io/repurpose/:

> Project Rephetio predictions are released under CC0 1.0. Compounds (identifiers, names, and desciptions) are from DrugBank, while diseases are from the Disease Ontology.

**Comment by dhimmel** [comment]
# disease genes tooltips

From https://het.io/disease-genes/browse/ Now at https://hetio.github.io/disease-genes-frontend/?tab=diseases&id=DOID_12236

- ID: disease ontology identifier
- Name: name of the disease
- Pathophysiology: our manual disease classification
- associations: the number of genes with a primary annotation to a high-confidence association that we extracted from the GWAS Catalog
- AUROC: area under the ROC curve indicating he model's ability to rank associated genes over unassociated genes for the specific disease.

**Comment by dhimmel** [comment]
Browse predictions of disease-gene associations from the study:

```
<p><strong>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</strong><br />
Daniel S. Himmelstein, Sergio E. Baranzini<br />
<em>PLOS Computational Biology</em> (2015-07-09) <a href="https://doi.org/98q">https://doi.org/98q</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1004259">10.1371/journal.pcbi.1004259</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26158728">26158728</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4497619">PMC4497619</a></p>
```

Our approach learns what types of paths in a heterogeneous network occur more frequently between genes and diseases that have been associated by GWAS. Using hetnet edge prediction we predict the probability that each gene associates with each disease.

Datasets related to this study are available at https://github.com/dhimmel/het.io-dag-data/tree/master/downloads.

**Comment by vincerubinetti** [comment]
closed by #15 and https://github.com/hetio/disease-genes-frontend/pull/7
